addon/components/mobile/flexberry-objectlistview.js

```diff
@@ -138,4 +140,159 @@ export default FlexberryObjectlistview.extend({
+        break;
+      default:
+        mobilePages.push(pages[2], pages[3], pages[4]);
+        break;
+    }
+
+    return mobilePages;
+  }),
+
+  /**
+    Array of pages without current.
+
+    @property _allPages
+    @type Array
+    @readOnly
+  */
+  _allPages: Ember.computed('pages', function() {
+    let allPages = [];
+    let mobilePagesNumbers = Ember.A();
+    let mobilePages = this.get('mobilePages');
+    mobilePages.forEach((page) => {
+      mobilePagesNumbers.push(page.number);
+    });
+
+    for (let i = 1; i <=  this.get('recordsTotalCount'); i++) {
+      if (!mobilePagesNumbers.contains(i)) {
+        allPages[i] = i;
+      }
+    }
+
+    return allPages;
+  }),
+
+  /*
+    Convert array of object sorting to array.
+
+    @property _currecntSortingArray
+    @type Array
+    @readOnly
+  */
+  _currecntSortingArray: Ember.computed('sorting',  function() {
+    let sorting = this.get('sorting');
+    let sortingKeys = Object.keys(this.get('sorting'));
+    let columns = Ember.A();
+    sortingKeys.forEach(key => {
+      let column = sorting[key];
+      columns.pushObject({
+        key: key,
+        sortNumber: column.sortNumber,
+        sortAscending: column.sortAscending
+      });
+    });
+    columns = columns.sort((a, b) => a.sortNumber - b.sortNumber);
+
+    return columns;
+  }),
+
+  /**
+    Class icons for sorting.
+
+    @property mobileSortingSettingsIcon
+    @type String
+    @readOnly
+  */
+  mobileSortingSettingsIcon: Ember.computed('sorting',  function() {
+    let icon = 'sort content descending';
+    let firstColumn = this.get('_currecntSortingArray')[0];
+
+    if (firstColumn !== undefined) {
+      icon = firstColumn.sortAscending ? 'sort content ascending' : 'sort content descending';
+    }
+
+    return `${icon} icon`;
+  }),
+
+  /**
+    Mobile sort text.
+
+    @property mobileSortingSettingsCaption
+    @type String
+    @readOnly
+  */
+  mobileSortingSettingsCaption: Ember.computed('sorting', 'i18n.locale',  function() {
+    let i18n = this.get('i18n');
+    let sorting = this.get('_currecntSortingArray');
+    if (sorting.length === 0) {
+      return i18n.t('components.flexberry-objectlistview.without-sorting');
+    }
+
+    let sortingValue;
+    sorting.forEach((column) => {
+      let columnHeader = i18n.t(getAttrLocaleKey(this.get('modelName'), this.get('modelProjection').projectionName, column.key)).string;
+      if (columnHeader !== undefined) {
+        let key = column.key.split('.')[0];
+        columnHeader = i18n.t(getAttrLocaleKey(this.get('modelName'), this.get('modelProjection').projectionName, key)).string;
+      }
+
+      if (sortingValue  !== undefined) {
+        sortingValue += `, ${columnHeader}`;
+      } else {
+        sortingValue = columnHeader;
+      }
+    });
+
+    return sortingValue;
+  }),
+
+  didRender: function() {
+    this._super(...arguments);
+    let _this = this;
+    let selectPageDropdown = Ember.$('.page-select-drodpown');
```

Comment:
Чтобы поиск элемента был в контексте элемента текущего компонента.
```suggestion
    let selectPageDropdown = this.$('.page-select-drodpown');
```